src/app/components/Header/index.tsx

```diff
@@ -54,7 +48,7 @@ export function Header() {
       };
     }
 
-    if (link.to.startsWith('http')) {
+    if (link.to.startsWith('http' || 'https')) {
```

Comment:
`.startsWith('http')` also includes `https` because it starts with `http` ;) Also, I don't think `||` works inside this method
```suggestion
    if (link.to.startsWith('http')) {
```

Reply:
ok